_includes/live-demos/inline/style.css

```diff
@@ -62,5 +62,5 @@
 }
 .demo-inline *[contentEditable="true"]:focus,
 .demo-inline *[contentEditable="true"]:hover {
-  outline: 2px solid #2276d2;
+  outline: 2px solid {{site.primaryColor}};
```

Comment:
Likewise this shouldn't be the same color.

Reply:
There is no border on the content, the border only appears on hover to indicate that you are hovering over an editor instance. Blue on white.